### DISCUSSION TITLE
3.1 release

### DISCUSSION THREAD

**Discussion by dopry** [django-oauth/django-oauth-toolkit#1511] [General]
@n2ygk How do you feel about me cutting a 3.1 release? I'd like to get the wildcard feature out there, partly because I need it.

**Comment by n2ygk** [comment]
It is kinda close to the last release. Can't you use a GitHub commit reference for the requirements.txt for the time being?

**Reply by dopry** [reply]
I can and am... I just don't see much reason to hold up a release. other people may want it... I'm pretty used to semantic-release deploying every merge on my other projects

**Comment by n2ygk** [comment]
Well since you are volunteering to do it, sure!

**Reply by dopry** [reply]
Also I want to go through the release workflow. I'll do it early next week.  I've got some client work to wrap up today.

**Reply by dopry** [reply]
doesn't look like I'm getting to it this week. got caught up with the logout confirmation screen issue. so maybe next week. :)

**Reply by n2ygk** [reply]
@dopry are you able to do this? I noticed that the Jazzband request to make you a lead has not happened yet...

**Reply by dopry** [reply]
I can probably pick this back up now. Waiting on pypi access still.

**Reply by n2ygk** [reply]
You have pypi access now.

**Reply by dopry** [reply]
Yep got it... Need to update the release.yml to publish... can you check out https://github.com/django-oauth/django-oauth-toolkit/pull/1592, it has the release workflow updates...

**Comment by 93578237** [comment]
Hi, github release for 3.1 is missing

**Reply by dopry** [reply]
yep. I need to update part of the CI pipeline. It's on pypi though.  If you want to take a stab at the CI pipeline you're welcome to.
